agent/roles/manifests/templates/install-config_yaml.j2

```diff
@@ -55,6 +55,29 @@ platform:
     platformName: oci
 {% elif (platform_type == "vsphere") %}
   vsphere:
+    vcenters:
```

Comment:
Now that there is more configuration for vsphere, recommend adding a new file `install-config_vsphere_yaml.j2 `and using it here similar to what we did for baremetal, based on @andfasano' suggestion - https://github.com/openshift-metal3/dev-scripts/blob/master/agent/roles/manifests/tasks/install-config.yml#L12-L16

Reply:
At some point though we'll have to make macros out of the common areas in these platform specific install-config files

Reply:
I created https://issues.redhat.com/browse/AGENT-746 to add macros to these platform specific install-config files.